django-server/server/api/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Cafeteria(models.Model):
+    name = models.CharField(max_length=128)
+    description = models.CharField(max_length=256)
+    sub_description = models.CharField(max_length=512)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+    capacity = models.IntegerField()
```

Comment:
Missing validator for minimum value as in API.

Reply:
to be discussed: cannot set constraint in django that will generate sql with needed checks on database, only write a validator that can check this value on server (fixed in this way). Have to check if this works with API generators